### PR TITLE
Update seo patterns

### PR DESCRIPTION
**Pull request by rdavid1099**
Add 'LA Devs - A Community for Los Angeles Based Software Developers' as the site title
Add temporary mission statement as meta description (A new home for Los Angeles based software developers is being built)
Add temporary LA Devs Favicon (using icon on Slack)

### CODE REVIEW

**Comment by j-sm-n** [comment]
Relates to: https://github.com/la-devs/issue-tracking/issues/9

File: src/pages/index.js [hunk 1]
```diff
@@ -3,14 +3,15 @@ import React from "react"
 import Layout from "../components/layout"
 import SEO from "../components/seo"
 import SlackInviteForm from "../components/slackInviteForm"
+import { SEO_TAGS } from '../utils/constants';
 
 const IndexPage = () => (
   <Layout>
-    <SEO title="Home" keywords={[`gatsby`, `application`, `react`]} />
+    <SEO title="LA Devs - A Community for Los Angeles Based Software Developers" keywords={SEO_TAGS} />
```
Comment (j-sm-n) [inline src/pages/index.js]: I may change this later--but side note, we can add another site title for any new pages we add. I think it's best practice to add a short description of the page folks are looking at. For example, if we add a code of conduct page, we should add a new site title that's something like, `Code of Conduct | LA Devs`.

File: src/pages/index.js [hunk 2]
```diff
@@ -3,14 +3,15 @@ import React from "react"
...
     <h1>Under Construction</h1>
-    <h4>A new home for software developers is being built</h4>
+    <h4>A new home for Los Angeles based software developers is being built</h4>
     <p>
-      Want to help us buid a community focused on helping each other connect, have fun with technology, make friends, and so much more?<br />
+      Want to help us buid an inclusive community focused on helping each other connect, having fun with technology, making friends, and so much more?<br />
```
Comment (j-sm-n) [inline src/pages/index.js]: 👍 🎉

File: src/utils/constants.js [hunk 3]
```diff
@@ -1,2 +1,3 @@
 export const EMAIL_REGEX = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
 export const LA_DEVS_EMAIL = 'info@ladevs.org';
+export const SEO_TAGS = ['los angeles', 'la', 'software', 'developers', 'networking', 'community'];
```
Comment (j-sm-n) [inline src/utils/constants.js]: Thinking about it now, we could probably add a `'slack'` keyword, too.

Comment (rdavid1099) [reply src/utils/constants.js]: I agree 👍

File: src/pages/index.js (hunk 1 above)
Comment (rdavid1099) [reply src/pages/index.js]: I totally agree. I'm sure we can set a bit of an automated process to handle titles on all pages and save a key-value pair in `constants.js`

**Pull request merged**